tools/install.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# read go packages out of ./tools.go
+go_packages=()
+while IFS='' read -r line; do
+  go_packages+=("$line")
+done < <(grep '^.*_ "' tools.go | sed 's/.*"\(.*\)"/\1/g')
+
+# $go_packages must be an array for the expansion here to work
+for tool in "${go_packages[@]}"; do
+  echo "  $tool"
+  go install "$tool"
+done
```

Comment:
You can use `go list` to avoid grep/sed.  Also, we can use xargs to avoid needing the "advanced" shell constructs.

What do you feel about something like:

```sh
go list --tags tools  -f '{{join .Imports "\n"}}' . | xargs --verbose go install -v
```

Reply:
Brilliant!